packages/website/blog/2023-03-13-announcing-typescript-eslint-v6-beta.md

```diff
@@ -0,0 +1,309 @@
+
+The biggest configuration change in typescript-eslint v6 is that we've reworked the names of our [provided user configuration files](https://typescript-eslint.io/linting/configs).
+typescript-eslint v5 provided three recommended configurations:
+
+- [`plugin:@typescript-eslint/recommended`](https://typescript-eslint.io/linting/configs#recommended): Recommended rules for code correctness that you can drop in without additional configuration.
+- [`plugin:@typescript-eslint/recommended-requiring-type-checking`](https://typescript-eslint.io/linting/configs#recommended-requiring-type-checking): Additional recommended rules that require type information.
+- [`plugin:@typescript-eslint/strict`](https://typescript-eslint.io/linting/configs#strict): Additional strict rules that can also catch bugs but are more opinionated than recommended rules.
+
+Those configurations worked well for most projects.
+However, some users correctly noted two flaws in that approach:
+
+- Strict rules that didn't require type checking were lumped in with those that did.
+- _Stylistic_ best practices were lumped in with rules that actually find bugs.
+
+As a result, we've reworked the configurations provided by typescript-eslint into these two groups:
+
+- Functional rule configurations, for best best practices and code correctness:
+  - **`plugin:@typescript-eslint/recommended`**: Recommended rules that you can drop in without additional configuration.
+  - **`plugin:@typescript-eslint/recommended-type-checked`**: Additional recommended rules that require type information.
+  - **`plugin:@typescript-eslint/strict`**: Additional strict rules that can also catch bugs but are more opinionated than recommended rules _(without type information)_.
+  - **`plugin:@typescript-eslint/strict-type-checked`**: Additional strict rules that do require type information.
+- Stylistic rule configurations, for consistent and predictable syntax usage:
+  - **`plugin:@typescript-eslint/stylistic`**: Stylistic rules you can drop in without additional configuration.
+  - **`plugin:@typescript-eslint/stylistic-type-checked`**: Additional stylistic rules that require type information.
+
+> `plugin:@typescript-eslint/recommended-requiring-type-checking` is now an alias for `plugin:@typescript-eslint/recommended-type-checked`.
+> The alias will be removed in a future major version.
+
+As of v6, we recommend that projects enable two configs from the above:
+
+- If you are _not_ using typed linting, enable `stylistic` and either `recommended` or `strict`, depending on how intensely you'd like your lint rules to scrutinize your code.
+- If you _are_ using typed linting, enable `stylistic-type-checked` and either `recommended-type-checked` or `strict-type-checked`, depending on how intensely you'd like your lint rules to scrutinize your code.
+
+For example, a typical project that enables typed linting might have an ESLint configuration file like:
+
+```js title=".eslintrc.cjs"
```

Comment:
It might be good to use the "remove this line", "add this line" syntax to highlight the necessary changes - WDYT?

Reply:
I dig it!

Reply:
I dig it!